spec/services/cost_determiner_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'rails_helper'
+
+RSpec.describe CostDeterminer do
+  let(:appeal_attrs) { {} }
+  let(:appeal) { double(appeal_attrs) }
+
+  subject { described_class.new(appeal) }
+
+  # TODO: Move this context to the appeal model specs, when we have some
+  context "when hmrc challenged is false" do
```

Comment:
I think we should change this one in the meantime to expect it to `raise` – as the appeal instance is not in a valid state for cost determination.